src/shared/inc/clrnt.h

```diff
@@ -1029,4 +1029,61 @@ RtlVirtualUnwind(
 
 #endif
 
+#ifdef TARGET_RISCV64
```

Comment:
Are all the files changed under `shared` come from the latest runtime repo which should have all the RISCV64 support needed?

Reply:
Yes, all this code from latest runtime repo only. Main point for this patch is move runtime repo risc-v related code, add crossbuild (libsos/libdbgshim/...) for risc-v. No other changes provided by this PR.